config/locales/client.en.yml

```diff
@@ -34,6 +35,13 @@ en:
       channel_close:
         title: "Close Channel"
         instructions: "Closing the channel prevents non-staff users from sending new messages or editing existing messages. Are you sure you want to close this channel?"
+      channel_delete:
+        title: "Delete Channel"
+        instructions: "<p>Deletes the <strong>%{name}</strong> channel and chat history. All messages and related data, such as reactions and uploads, will be permanently deleted. If you want to preserve the channel history and decomission it, you may want to archive the channel instead.</p>
+        <p>Are you sure you want to <strong>permanently delete</strong> the channel? To confirm, type the name of the channel in the box below.</p>"
```

Comment:
I think this was briefly mentioned in dev but should we be including the channel name in this message?

Reply:
Yes, we do already? "Deletes the <strong>%{name}</strong> channel"

Reply:
Ahh I didn't catch this when I was looking at the screenshot. I feel like it isn't as clear as what Github does but we can always try this out first and refine later.